test/blink/2-keyframes-with-offsets.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<script src="testharness/testharness.js"></script>
-<script src="testharness/testharnessreport.js"></script>
+<script src="../web-platform-tests/resources/testharness.js"></script>
```

Comment:
Yes except for that last one. We can't have /resources/ pointing to two different places, it can just remain a relative path without any issues.